### DISCUSSION TITLE
I have problem ERROR `/assets/js/dist/home.min.js' not found.

### DISCUSSION THREAD

**Discussion by kangminhyuk1111** [cotes2020/jekyll-theme-chirpy#1257] [Q&A]
### Checklist

- [X] I'm willing to follow the [Contributing Guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

ERROR `/assets/js/dist/home.min.js' not found.

### Operations you have already tried

Please give me a solution

I just did a fork, but it came out like this

**Comment by fabse-hack** [comment]
You solve the problem?

I think is the local building in the building and deploy yml.
Greetings

**Comment by Vicsolorz** [comment]
I have the same problem.

**Comment by wiliamvj** [comment]
> You solve the problem?
> 
> I think is the local building in the building and deploy yml. Greetings



> ### Checklist
> * [x]  I'm willing to follow the [Contributing Guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).
> 
> ### How did you create the site?
> Generated from `chirpy-starter`
> 
> ### Description
> ERROR `/assets/js/dist/home.min.js' not found.
> 
> ### Operations you have already tried
> Please give me a solution
> 
> I just did a fork, but it came out like this
> 
> ### Anything else?
> _No response_

You need to run `bash tools/init`

**Comment by joshjohanning** [comment]
https://github.com/cotes2020/jekyll-theme-chirpy/wiki/Upgrade-Guide#upgrade-the-fork

> Starting with v5.6.0, the JS distribution files have been removed from the repository, so for all future upgrades, you should compile the JS files yourself.
> 
> ```sh
> npm run build
> ```
> And then make sure to add them to your repository files.
> 
> ```sh
> git add assets/js/dist -f
> ```


I run this all as one command when I update my theme

```bash
npm run build && git add assets/js/dist -f && git commit -m "update js assets"
```
